packages/web-components/src/components/cbp-toggle/cbp-toggle.stories.tsx

```diff
@@ -0,0 +1,144 @@
+            control: 'object',
+        },
+    }
+  };
+
+const Template = ({label, checked, hideStatus, statusTexton, statusTextoff, disabled, context, sx }) => {
+    return ` 
+        <cbp-toggle
+            ${checked ? 'checked' : ''}
+            ${hideStatus ? `hide-status` : ''}
+            ${statusTexton ? `status-text-on="${statusTexton}"` : ''} 
+            ${statusTextoff ? `status-text-off="${statusTextoff}"` : ''}
+            ${disabled ? 'disabled' : ''}
+            ${context && context != 'light-inverts' ? `context=${context}` : ''}
+            ${sx ? `sx=${JSON.stringify(sx)}` : ``}
+            >
+            <label
+                slot='toggleLabel'
+            >
+                ${label}
+            </label>
+            <input
+                type="checkbox"
+                name="checkbox"
+                value="1"
+            />
+        </cbp-toggle>
+    `;
+};
+  
+export const Toggle = Template.bind({});
+  
+Toggle.args = {
+    label: 'Toggle Label:'
+}
+Toggle.argTypes = {
+    label: {
+        description: 'Sets the label for the toggle control',
+        control:'text'
+    },
+    checked: {
+        description: 'Sets the state of the toggle',
+        control: 'boolean'
+    },
+    statusTexton: {
+        description: 'Sets the label for the `on` state of the toggle control',
+        control: 'text'
+    },
+    statusTextoff: {
+        description: 'Sets the label for the `off` state of the toggle control',
+        control: 'text'
+    },
+}
+
+function generateToggles(items, labelWidth, hideStatus, disabled, context, sx){
+    const html = items.map(({label, statusTexton, statusTextoff}) => {
+    return `<cbp-toggle
+        ${hideStatus ? `hide-status` : ''}
+        ${statusTexton ? `status-text-on="${statusTexton}"` : ''} 
+        ${statusTextoff ? `status-text-off="${statusTextoff}"` : ''}
+        ${disabled ? 'disabled' : ''}
+        ${context && context != 'light-inverts' ? `context=${context}` : ''}
+        ${sx ? `sx=${JSON.stringify(sx)}` : ``}
+        >
+            <label
+                slot='toggleLabel'
+                style='width: ${labelWidth}'
+            >
+                ${label} 
+            </label>
+            <input
+                type="checkbox"
+                name="checkbox"
+                value="1"
+            />
+        </cbp-toggle>
+        `;
+    });
+    return html.join('');
+  }
+
+const MultipleTemplate = ({ToggleItems, labelWidth, hideStatus, disabled, context, sx }) => {
+    return `
+    <div>
+        ${generateToggles(ToggleItems, labelWidth, hideStatus, disabled, context, sx )} 
+    </div>
+    `;
+}
+
+export const MultipleToggle = MultipleTemplate.bind({});
+
+MultipleToggle.args={
+    ToggleItems: [
+        {
+            label: 'Toggle #1:',
+            statusTexton: '#1 On',
```

Comment:
I thought we said get rid of these status text overrides and just use the main story controls, so they are all the same. It is the most likely use case.

Reply:
uploaded what you saw in discussion earlier, addresses this issue